ros_rate: 20
sub_imu_topic: /mavros/imu/data
sub_odometry_topic: /mavros/optical_flow
sub_image_topic: /stereo/left/image_rect
pub_detected_circles_image: /detected_circles

#constants for the Hungarian Algo
threshold_pixel_distance: 100
